### PR TITLE
DCOS-8098: Add log link to task table

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Add an icon that links directly to logs tab.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @jfurrow

File: src/js/components/TaskTable.js [hunk 1]
```diff
@@ -145,6 +158,27 @@ class TaskTable extends React.Component {
...
   }
 
+  renderLog(prop, task) {
+    let title = task.name || task.id;
+    let params = this.props.parentRouter.getCurrentParams();
+    let routeParams = Object.assign({taskID: task.id}, params);
+
+    let linkTo = 'services-task-details-logs';
+    if (params.nodeID != null) {
+      linkTo = 'nodes-task-detailss-logs';
```
Comment (pierlo-upitup) [inline src/js/components/TaskTable.js]: `detailss`?

**Comment by pierlo-upitup** [comment]
Except for that one extra `s`, LGTM 🚢

**Comment by pierlo-upitup** [comment]
feel free to merge once it goes back to green

**Pull request merged**